tools/docker/builder/openwrt/build.sh

```diff
@@ -82,7 +85,7 @@ main() {
     while true; do
         case "$1" in
             -h | --help)               usage; exit 0; shift ;;
-            -b | --build-options)      BUILD_OPTIONS="$2"; shift; shift ;;
+            -b | --build-options)      BUILD_OPTIONS+=($(echo -n "$2")); shift; shift ;;
```

Comment:
Here I think the safest is to drop '--build-options' entirely. I think nobody use it anymore.

```suggestion
```

Reply:
Ok, i'll do that if almost one more reviewer will approve your suggestion

Reply:
I agree!

Reply:
Than drop it is. This option is removed.